specification/servicelinker/ServiceLinker.Management/back-compatible.tsp

```diff
@@ -0,0 +1,82 @@
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+using Microsoft.ServiceLinker;
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(DryrunPatch.properties);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(LinkerPatch.properties);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(ValidateOperationResult.properties);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(ConfigurationNameItem.properties);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
```

Comment:
Tooling should provide a better message for suppressions when we use the flattenProperty decorator. We already know that this is for back compat with client libraries, that should be a good message to put as justification